.drone.yml

```diff
@@ -25,7 +25,7 @@ pipeline:
     image: lampepfl/dotty:2017-10-20
     commands:
       - cp -R . /tmp/1/ && cd /tmp/1/
-      - ./project/scripts/sbt ";compile ;testAll ;dotty-bench/jmh:run 1 1 tests/pos/alias.scala"
+      - ./project/scripts/sbt ";compile ;testAll ;dotty-bench/jmh:run 1 1 tests/pos/alias.scala; ;dotc tests/run/arraycopy.scala ;dotr Test"
```

Comment:
`tests/pos/alias.scala; ;dotc` double semicolon?

Reply:
Also not sure it is the best place to test an sbt task. @smarter Do you think it should be a scripted test?

Reply:
Scripted tests are always good in my opinion :)